erts/emulator/drivers/common/inet_drv.c

```diff
@@ -517,6 +517,13 @@ static typeof(sctp_freepaddrs) *p_sctp_freepaddrs = NULL;
 static void (*p_sctp_freepaddrs)(struct sockaddr *addrs) = NULL;
 #endif
 
+#if defined(__GNUC__) && defined(HAVE_SCTP_CONNECTX)
+static typeof(sctp_connectx) *p_sctp_connectx = NULL;
+#else
+static int (*p_sctp_connectx)
+        (int sd, struct sockaddr * addrs, int addrcnt, sctp_assoc_t * assoc_id) = NULL;
+#endif
+
```

Comment:
Shouldn't this be:
``` C
#ifdef HAVE_SCTP_CONNECTX
#ifdef __GNUC__
// GNU typeof declaration of sctp_connectx
#else
// Fallback declaration of sctp_connectx
#endif
#endif

Reply:
Nope.  It is just like for the other functions.  Looks fine.

Reply:
Yes, I have to admit I copied the pattern from `sctp_bindx` without thinking about it too hard!